setup.py

```diff
@@ -99,7 +104,54 @@ def gen_packages_items():
     return packages
 
 
+def add_mim_extention():
+    """Add extra files that are required to support MIM into the package.
+
+    These files will be added by creating a symlink to the originals if the
+    package is installed in `editable` mode (e.g. pip install -e .), or by
+    copying from the originals otherwise.
+    """
+
+    # parse installment mode
+    if 'develop' in sys.argv:
+        # installed by `pip install -e .`
+        mode = 'symlink'
+    elif 'sdist' in sys.argv or 'bdist_wheel' in sys.argv:
+        # installed by `pip install .`
+        # or create source distribution by `python setup.py sdist`
+        mode = 'copy'
+    else:
+        return None
+
+    item_list = ['tools', 'configs', 'demo', 'model-index.yml']
+    os.makedirs('mmpose/.mim', exist_ok=True)
+
+    for item in item_list:
+        if osp.exists(item):
+            tar_path = osp.join('mmpose/.mim', item)
```

Comment:
item renames to filename
```suggestion
            tar_path = osp.join(mim_root, item)
```